frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -673,6 +675,16 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxWinners: Get<u32>;
 
+		/// Something that calculates the signed deposit base based on the signed submissions queue
+		/// size.
+		type SignedDepositBase: DepositCalculator<BalanceOf<Self>>;
```

Comment:
What is `SignedFixedDepositBase` and `SignedDepositBase`?

Reply:
`SignedDepositBase` is the type that implements the final signed deposit base (i.e the `DepositCalculator` implementor). `SignedFixedDepositBase`, on the other hand, is the initial term of the progression (thus "Fixed").

Simplifying, we have:

```
fn signed_deposit_base(n_term: usize) -> Balance {
  signed_fixed_deposit_base * (1 + increase_factor)^n_term
} 
```

I agree that the naming is not ideal though. I think that with the simplifications that you suggested above, we can improve on the naming as well.

Reply:
I improved the naming, I think it's looking better and more clear now.